MetaMask/metamask-extension
# add bth icoin logo

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28672?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Click on network picker and click on add custom network
2. ADD name "BTH"; RPC: https://rpc.bthscan.io and chainId 14149
3. Click on add network
4. You should see network icon

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
